test/integration/tests/fei.ts

```diff
@@ -2,8 +2,8 @@ import chai, { expect } from 'chai';
 import CBN from 'chai-bn';
 import { solidity } from 'ethereum-waffle';
 import { ethers } from 'hardhat';
-import { NamedContracts } from '@custom-types/types';
-import { expectRevert, resetFork, ZERO_ADDRESS } from '@test/helpers';
+import { NamedAddresses, NamedContracts } from '@custom-types/types';
```

Comment:
Good time to try out forge unit tests imo

Reply:
oof I def should have done this